Regression/Checksum/update_benchmarks_from_azure_output.py

```diff
@@ -0,0 +1,48 @@
+import re
+import sys
+
+azure_output_filename = sys.argv[1]
+
+first_char = 29 # first non prefix character
+test_name_str = "working on test:"
+test_name_first_char = 50 # start of test name
+test_name_last_char = -5 # end of test name
+plotfile_str = "Plotfile : "
+benchmark_path = "benchmarks_json/"
+benchmark_suffix = ".json"
+
+current_test = ""
+
+f = open(azure_output_filename, "r")
+
+for line in f:
+    if (line.find(test_name_str)!= -1):
+        current_test = line[test_name_first_char:test_name_last_char]
+    line_noprefix = line[first_char:]
+    if (line_noprefix.startswith(plotfile_str)):
```

Comment:
Here we could add a comment like the following:
```suggestion
    # Here we parse lines that look like this:
    # "Plotfile : [lev=0,jx] 1.012345678901234e+16"
    if (line_noprefix.startswith(plotfile_str)):
```